### ISSUE TITLE
Move table discordant with basecalls

### ISSUE THREAD

**Issue by Joshmoss11** [nanoporetech/remora#69]
I have attempted to run remora on some custom data.
Remora (2.1) was run with the following parameters:
remora dataset prepare can.pod5 can_aligned.bam --output-remora-training-file can_chunks.npz  \
--mod-base-control --log-filename can_small.log

I receive output:
Indexing BAM by read id: 10000 Reads [00:00, 114296.49 Reads/s]
[11:33:00] Extracting read IDs from POD5
[11:33:00] Found 10000 BAM records, 1600 POD5 reads, and 1600 in common
[11:33:00] Making reference-anchored training data
[11:33:00] Allocating memory for output tensors
[11:33:00] Processing reads
Extracting chunks: 100%|█████████████████████████████████████████████████████| 1600/1600 [00:01<00:00, 1452.70 Reads/s]
[11:33:01] Unsuccessful read/chunk reasons:
   1600 : Move table discordant with basecalls
[11:33:02] Extracted 0 chunks from 1600 reads.
[11:33:02] Label distribution: Counter()

Looking in the log file for each read I see some variation of:
DEBUG [11:33:00:AddAlignments_0:MainThread:io.py:318] Move table (num moves: 230) discordant with basecalls (seq len: 104)

Basecalling was done with guppy with parameters:

guppy_basecaller -i pod5 --recursive -s out --barcode_kits SQK-NBD114-24 \
        --trim_adapters --records_per_fastq 10000 --do_read_splitting --gpu_runners_per_device 16 \
        --cpu_threads_per_caller 2 --device cuda:all --bam_out --moves_out \
        --detect_adapter --detect_barcodes --enable_trim_barcodes --config dna_r10.4.1_e8.2_400bps_sup.cfg

So, in summary, no chunks are extracted from the reads. This seems to be attributed to a discordance between the basecalls and the move table. 
I suspect that this has to do with the fact that the barcodes were trimmed from the reads during basecalling?
Is that likely?
If yes, how else would you suggest trimming the barcodes?
If not, any idea what the problem is?
Thanks!

**Comment by marcus1487** [comment]
The barcode trimming is almost certainly to blame for this issue. We working on solutions for this, but there is not currently the any available software that can clip barcodes and leave valid move tables. 

As a workaround for the time being, would it be possible to continue processing with the barcodes included in the reads? Dataset preparation will only include data mapped to a reference, so the barcodes are unlikely to effect the output dataset much if at all.

P.S. I would also recommend setting the `--motif` option as the default which is all bases is likely not correct for most all settings. We could consider making the parameter required in a future version.

**Comment by Joshmoss11** [comment]
Indeed. Dataset prep works smoothly without barcode/adapter trimming.
Thanks for the --motif tip.
